src/go/intelligence/int_test/test_config.json

```diff
@@ -66,8 +66,19 @@
       },
       "annotationMap": {
         "label": "shortDescription",
-        "actions": ["intervention", "outage"],
-        "systems": ["network", "database", "db"],
+        "annotations": [
+          {
+            "actions": ["intervention", "outage"],
+            "systems": ["network", "database", "db"],
+            "tags": ["cmsweb-play"]
+          },
+
+          {
+            "actions": ["update", "upgrade"],
+            "systems": ["network", "database", "db"],
+            "tags": ["cmsweb-play2"]
+          }
+        ],
         "urlLabel": "URL"
```

Comment:
I think urlLabel will be part of each annotation since it may refer to different URLs, e.g. one for SSB and another for GGUS. Moreover. do you really need to call it urlLabel, it represents the URL and I think it should be called URL in the configuration.

Reply:
Valentin, yes we need to call it urlLabel because in alert data it's a field "URL", so if there's change in the alerting services in future in case, we can change the urlLabel value here accordingly. Moreover, each service has unique urlLabel identifier, that said we don't need to add multiple times in the annotations array above. The above example is of one service, if you check the test_config.json you can see I have maintained different urlLabel for different services.